nih-sparc/sparc-app
# Organ details page - Display header data #4a9h38

# Description

The purpose of this PR is to display the data for an organ in the header.

## Type of change


# How Has This Been Tested?

- Visit the Find Data page
- Click on the 'Organs' tab
- Click on the 'Heart' organ
- You should be taken to that organ's page
- The data in the header should be populated

# Checklist:
